screen/transactions/src/main/java/com/ivy/transactions/TransactionsViewModel.kt

```diff
@@ -335,11 +341,21 @@ class TransactionsViewModel @Inject constructor(
             currency.value = initialAccount.currency!!
         }
 
-        val account = account.value
+        val account = com.ivy.data.model.Account(
+            AccountId(initialAccount.id),
+            NotBlankTrimmedString(initialAccount.name),
```

Comment:
That's expected. Also use `NotBlankTrimmedString.from(...)` and prevent adding the account when the name is blank

Reply:
`NotBlankTrimmedString.from(...)` returns Either<X,Y> but we're not in a raise context

Reply:
That's the point - use `either { }` or `.getOrNull()` the ViewModel must handle the case when then account name is blank. https://arrow-kt.io/learn/typed-errors/